src/lib.rs

```diff
@@ -518,6 +814,13 @@ mod tests {
     fn max_level_works() {
         let logger = Logger::new().max_level(LogLevel::Trace);
         assert_eq!(logger.level, LogLevel::Trace);
+        assert!(logger.verbosity.is_none());
+    }
+
+    #[test]
+    fn base_level_works() {
+        let logger = Logger::new().base_level(LogLevel::Info);
+        assert_eq!(logger.offset, 2);
```

Comment:
maybe have an assert that `init_with_verbosity` here giving the warn level

Reply:
Are you suggesting a new test or an assert macro within this test?